src/xmipp/libraries/data/ctf.cpp

```diff
@@ -1205,19 +1204,18 @@ void CTFDescription::readFromMdRow(const MDRow &row, bool disable_if_not_K)
 
 void CTFDescription::readFromMetadataRow(const MetaData &md, size_t id, bool disable_if_not_K)
```

Comment:
is this copy & paste version of the CTFDescription1D?

Reply:
Sorry, I don't know. I didn't touch that code.